src/contracts/BPool.sol

```diff
@@ -505,13 +491,16 @@ contract BPool is BToken, BMath, IBPool {
     }
     Record storage inRecord = _records[tokenIn];
     Record storage outRecord = _records[tokenOut];
-    return calcSpotPrice(
+
+    spotPrice = calcSpotPrice(
       IERC20(tokenIn).balanceOf(address(this)),
       inRecord.denorm,
       IERC20(tokenOut).balanceOf(address(this)),
       outRecord.denorm,
       _swapFee
     );
+
+    return spotPrice;
```

Comment:
needed?

Reply:
is not needed, yet it was chosen to maintain consistency across methods, most of them already explicitly returned the calculated variable.

we went for:
- no naming return var in very simple returns (e.g. the `return true` in approvals)
- naming return var and explicitly returning it (to maintain code clarity)

this came handy for example in the Helper, that complying to an interface forced us to include a return var that was unused, then we could have not named it (with solidity warnings) or name it and return it empty (no warnings)